### PR TITLE
Move injured passengers to incident report

### LINKED ISSUE #126: Put injured passengers on incident report
Rather than supervisor report.

### PR DESCRIPTION
**Pull request by dfaulken**
Requested by Carolyn, Tom, and Liza. Closes #126.

### CODE REVIEW

File: spec/system/staff/viewing_incidents_spec.rb [hunk 1]
```diff
@@ -7,8 +7,8 @@
   before(:each) { when_current_user_is staff }
   context 'searching by claim number' do
     it 'allows searching for incidents by claim number' do
-      incident = incident_in_divisions(staff.divisions, claim_number: 'apples')
-      incident = incident_in_divisions(staff.divisions, claim_number: 'bananas')
+      incident_in_divisions(staff.divisions, claim_number: 'apples')
+      incident_in_divisions(staff.divisions, claim_number: 'bananas')
```
Comment (mboneil10) [inline spec/system/staff/viewing_incidents_spec.rb]: So fruity.

File: app/views/incident_reports/_form.html.haml [hunk 2]
```diff
@@ -244,6 +244,32 @@
         prompt: true
   .passenger-incident-info{style: ('display: none;' unless report.passenger_incident?)}
     %h3 Passenger Incident Information
+    .field
+      = form.label :inj_pax_info
+      = form.check_box :inj_pax_info, checked: report.has_injured_passengers?,
+        id: :supervisor_report_inj_pax_info
+    .inj-pax-info{style: ('display: none;' unless report.has_injured_passengers?)}
+      %h3 Injured Passenger Info
```
Comment (mboneil10) [inline app/views/incident_reports/_form.html.haml]: So, I didn't fill anything in on the Driver Account of Incident. I just submitted it, and as expected, errors were displayed. However, when I checked `Did the incident involve a passenger?`, `Were passengers injured?`, left `Injured Passenger Info` blank, and clicked `Save report and print PDF`, the errors don't appear anymore.

Didn't look too far into it, just wanted to let you know that this doesn't seem like expected behavior to me. Could be wrong.

File: app/views/incident_reports/_form.html.haml [hunk 3]
```diff
@@ -244,6 +244,32 @@
...
+            = pax_form.label :transported_to_hospital
+            = pax_form.check_box :transported_to_hospital
+          .field
+            = pax_form.label :home_phone
+            = pax_form.text_field :home_phone
+            = pax_form.label :cell_phone
+            = pax_form.text_field :cell_phone
+            = pax_form.label :work_phone
+            = pax_form.text_field :work_phone
+      %button.btn-default.add-pax Add injured passenger info
```
Comment (mboneil10) [inline app/views/incident_reports/_form.html.haml]: Is there a way to delete these entries? I clicked this button a lot and bulked up the page.

File: app/views/incident_reports/_form.html.haml [hunk 4]
```diff
@@ -244,6 +244,32 @@
         prompt: true
   .passenger-incident-info{style: ('display: none;' unless report.passenger_incident?)}
     %h3 Passenger Incident Information
+    .field
```
Comment (mboneil10) [inline app/views/incident_reports/_form.html.haml]: A nitpick, but the placement of this is a bit weird to me. I would expect the accordion to be exactly below the `Did the incident involve a passenger?`.

File: app/models/incident_report.rb [hunk 5]
```diff
@@ -63,6 +66,10 @@ def full_location(include_state: false)
     parts.join ', '
   end
 
+  def has_injured_passengers?
```
Comment (mboneil10) [inline app/models/incident_report.rb]: Looks good

File: app/views/incident_reports/_form.html.haml (hunk 3 above)
Comment (dfaulken) [reply app/views/incident_reports/_form.html.haml]: Opened https://github.com/umts/incidents/issues/133; will resolve outside of this PR.

File: app/views/incident_reports/_form.html.haml (hunk 2 above)
Comment (dfaulken) [reply app/views/incident_reports/_form.html.haml]: Something strange is going on. In my environment, when I enter any passenger info and submit the form, the submit button is no longer clickable. Pressing Enter in a text field doesn't submit the form. Will investigate more tomorrow.

File: app/views/incident_reports/_form.html.haml (hunk 4 above)
Comment (dfaulken) [reply app/views/incident_reports/_form.html.haml]: Agreed. It was based on the PDF form, but for a web form the layout isn't intuitive. Opened https://github.com/umts/incidents/issues/134.

File: app/views/incident_reports/_form.html.haml (hunk 2 above)
Comment (dfaulken) [reply app/views/incident_reports/_form.html.haml]: Fixed. https://github.com/umts/incidents/pull/128/commits/04dc98fab1b12cacbac39379aa6b5b9d39ddcce4

Comment (mboneil10) [reply app/views/incident_reports/_form.html.haml]: Yup, looks good.

**Pull request merged**